PeerioTechnologies/peerio-desktop
# chore: replace RTB <Input>, final RTB cleanup

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/1340/desktop-replace-input-components
https://app.clubhouse.io/peerio/story/2834/peerui-final-cleanup 

#### Testing instructions  
1. Changes to `package.json` -> verify that it actually works for you! I rm'd all node_modules and installed again, works for me, but plz check on your side too :)
2. Then can test on `dev` as with other RTB replacements

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
